src/client/game.cpp

```diff
@@ -4217,6 +4193,35 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats, f32 dtime,
 	g_profiler->avg("Game::updateFrame(): update frame [ms]", tt_update.stop(true));
 }
 
+void Game::updateClouds(float dtime)
+{
+	if (this->sky->getCloudsVisible()) {
+		this->clouds->setVisible(true);
+		this->clouds->step(dtime);
+		// camera->getPosition is not enough for 3rd person views
+		v3f camera_node_position = this->camera->getCameraNode()->getPosition();
+		v3s16 camera_offset      = this->camera->getOffset();
+		camera_node_position.X   = camera_node_position.X + camera_offset.X * BS;
+		camera_node_position.Y   = camera_node_position.Y + camera_offset.Y * BS;
+		camera_node_position.Z   = camera_node_position.Z + camera_offset.Z * BS;
+		this->clouds->update(camera_node_position,
+				this->sky->getCloudColor());
+		if (this->clouds->isCameraInsideCloud() && this->m_cache_enable_fog) {
+			// if inside clouds, and fog enabled, use that as sky
+			// color(s)
```

Comment:
This line shouldn't be wrapped either.

Reply:
I actually meant the line above (the comment) :)

Reply:
It looks to me like the line is not wrapped.

Reply:
I interpreted this to mean

```
// if inside clouds, and fog enabled, use that as sky color(s)
```

Since the original code was also wrapped way too short in other places, the author could have meant it either way. Either way is fine. Marking this as resolved.